modules/ROOT/pages/type-definitions/directives/custom-logic.adoc

```diff
@@ -0,0 +1,641 @@
+:description: This page describes how to use directives for custom logic.
+:page-aliases: type-definitions/cypher.adoc, type-definitions/default-values.adoc, ogm/examples/custom-resolvers.adoc, custom-resolvers.adoc
+= Custom logic
+
+== `@cypher`
+
+The `@cypher` directive binds a GraphQL field to the results of a Cypher query.
+This directive can be used both for properties in a type or as top level queries.
+
+=== Global variables
+
+Global variables are available for use within the Cypher statement and can be applied to the `@cypher` directive.
+
+[cols="1,2,2"]
+|===
+| Variable | Description | Example
+
+| `this`
+| This value is a reference to the currently resolved node, and it can be used to traverse the graph.
```

Comment:
In order to avoid the repetition of "this value" in the following item:

```suggestion
| Refers to the currently resolved node, and can be used to traverse the graph.
```